cmd/security-agent/app/app.go

```diff
@@ -181,3 +197,76 @@ func start(cmd *cobra.Command, args []string) error {
 	log.Flush()
 	return nil
 }
+
+func startCompliance(stopper restart.Stopper) error {
```

Comment:
seems like we're duplicating some code from `cmd/security-agent/app/compliance.go` in this function, is it possible to do some refactoring?

Reply:
yes, definitely, thank you